packages/sdk-lsk/source/fee.service.ts

```diff
@@ -32,6 +32,7 @@ export class FeeService extends Services.AbstractFeeService {
 			htlcLock: this.#transform(0),
 			htlcClaim: this.#transform(0),
 			htlcRefund: this.#transform(0),
+			unlockToken: this.#transform(0.1 * 1e8),
```

Comment:
What is this number based on?

Reply:
:hide: what are other numbers based on? I feel like we could drop this PR given we'll be using`calculate` for unlock type. What do you think?

Reply:
The other values are static fees from 2.0 but 3.0 no longer has static fees.

Reply:
This can be changed to 0 because there's no static fee, calculation is required.

Reply:
I see, thanks. I'll update the tests and push.